onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -65,22 +150,32 @@ def augment_graph(self):
         tensors_to_calibrate = set()
 
         for node in model.graph.node:
-            should_be_calibrate = ((node.op_type in self.calibrate_op_types) and
-                                   (node.name not in self.black_nodes)) or (node.name in self.white_nodes)
+            if augment_all_ops:
+                should_be_calibrate = True
+            else:
+                should_be_calibrate = ((node.op_type in self.calibrate_op_types) and
+                                       (node.name not in self.black_nodes)) or (node.name in self.white_nodes)
             if should_be_calibrate:
                 for tensor_name in itertools.chain(node.input, node.output):
                     if tensor_name in value_infos.keys():
```

Comment:
Will be shape inference done first on the model in order to have all tensor names included in value_infos? If it is, we need to mentioned it in the document, otherwise user won't get completed calibration table by using original model.

Reply:
Okay, I will update to document.

Reply:
Document updated.